base/linalg/lapack.jl

```diff
@@ -94,6 +100,17 @@ function chkfinite(A::StridedMatrix)
     return true
 end
 
+# LAPACK version number
+function laver()
+    major = BlasInt[0]
+    minor = BlasInt[0]
+    patch = BlasInt[0]
+    ccall((@blasfunc(ilaver_), liblapack), Void,
+        (Ptr{BlasInt}, Ptr{BlasInt}, Ptr{BlasInt}),
+        major, minor, patch)
+    return major[1], minor[1], patch[1]
+end
```

Comment:
Although this function is only used during initialization time, maybe it's better to just return a `VersionNumber` from this function? Or is a tuple easier in some cases? Also I think allocating a `Ref{BlasInt}()` is faster than `BlasInt[0]`.